tools/projmgr/src/ProjMgrYamlEmitter.cpp

```diff
@@ -596,6 +711,9 @@ bool ProjMgrYamlBase::WriteFile(YAML::Node& rootNode, const std::string& filenam
       ProjMgrLogger::Error(filename, "file cannot be written");
       return false;
     }
+    if (!schemaName.empty()) {
+      fileStream << "# yaml-language-server: $schema=https://raw.githubusercontent.com/Open-CMSIS-Pack/devtools/main/tools/projmgr/schemas/" << schemaName << ".schema.json" << endl << endl;
+    }
```

Comment:
The `# yaml-language-server` comment has higher priority over IDE workspace and user selections. To avoid hiding such selections it's recommended not to print this comment by default.